lms/routes.py

```diff
@@ -50,6 +50,11 @@ def includeme(config):
         "blackboard_api.files.via_url", "/api/blackboard/courses/{course_id}/via_url"
     )
 
+    config.add_route(
+        "onedrive.filepicker.authorize",
+        "/api/onedrive/filepicker/authorize",
+    )
```

Comment:
```suggestion
    config.add_route(
        "onedrive.filepicker.authorize", "/api/onedrive/filepicker/authorize"
    )
```

The trailing comma forces Black to wrap each item onto its own line. If you remove the trailing comma it'll fit them both on one line

Reply:
I think `"onedrive.filepicker.redirect"` might be a better name for this route, so it's the OneDrive redirect URL rather than an authorize URL.

For the path I think maybe just `"/onedrive/filepicker/redirect"` since it's an HTML page not an API endpoint.